### PR TITLE
hexedit: Add new package

### PR DESCRIPTION
**Pull request by WereCatf**
Viewer and editor in hexadecimal or ASCII for files or devices

Signed-off-by: Nita Vesa <werecatf@outlook.com>

Maintainer: me
Compile tested: ath79, ramips, amd64, OpenWrt 21.02.3 and ipq40xx on trunk
Run tested: ath79 and amd64 on OpenWrt 21.02.3, ipx40xx on trunk

### CODE REVIEW

File: utils/hexedit/Makefile [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2007-2022 OpenWrt.org
```
Comment (jefferyto) [inline utils/hexedit/Makefile:2]: Either omit this line or assign copyright to yourself.

File: utils/hexedit/Makefile [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hexedit
+PKG_VERSION:=1.6
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/pixel/hexedit.git
+PKG_SOURCE_VERSION:=eab92dcaa34b66bc5182772afc9fda4ac8a27597
```
Comment (jefferyto) [inline utils/hexedit/Makefile:16]:
```suggestion
PKG_SOURCE_VERSION:=$(PKG_VERSION)
```

To checkout the tag directly.

File: utils/hexedit/Makefile [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+PKG_MIRROR_HASH:=138f8aff0466a33f33be43e25e7a47ba48216c0b2d41cbb6cc23ca848bd8a564
+
+PKG_LICENSE:=GPL-2.0-only
```
Comment (jefferyto) [inline utils/hexedit/Makefile:19]:
```suggestion
PKG_LICENSE:=GPL-2.0-or-later
```

Based on [code comment](https://github.com/pixel/hexedit/blob/eab92dcaa34b66bc5182772afc9fda4ac8a27597/hexedit.c#L6-L7).

File: utils/hexedit/Makefile [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  or its partitions, flash drives, etc. These elements will not be wholly read
+  because hexedit loads a piece at a time.
+
+  hexedit can be used by programmers in several activities. It also is useful
+  for debugging, forensics investigations and recovering from digital disasters.
+endef
+
+define Package/hexedit/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(CP) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
```
Comment (jefferyto) [inline utils/hexedit/Makefile:56]:
```suggestion
	$(CP) $(PKG_INSTALL_DIR)/usr/bin/hexedit $(1)/usr/bin/
```

For better readability.

**Review by jefferyto — CHANGES_REQUESTED** [review]

File: utils/hexedit/Makefile (hunk 2 above)
Comment (WereCatf) [reply utils/hexedit/Makefile]: Didn't know it could be done like that directly, thanks.

**Review by jefferyto — CHANGES_REQUESTED** [review]

**Comment by jefferyto** [comment]
Thanks!

**Pull request merged**